api/python/arrus/utils/tests/utils.py

```diff
@@ -175,8 +175,8 @@ def get_default_context(self, device=None, sequence=None, medium=None):
                     pitch=0.2e-3,
                     curvature_radius=0.0
                 ),
-                sampling_frequency=65e6,
-                data_sampling_frequency=65e6
+                sampling_frequency=120e6,
+                data_sampling_frequency=120e6
```

Comment:
Sampling frequency should depend on AFE(OEM) variant

Reply:
I think this is acceptable, as this is an example (default) value used by the image reconstruction unit tests.